src/Table/TableBody.js

```diff
@@ -119,29 +120,25 @@ const TableBody = React.createClass({
      * Override the inline-styles of the root element.
      */
     style: React.PropTypes.object,
-  },
-
-  contextTypes: {
+  };
+
+  static defaultProps = {
+    allRowsSelected: false,
+    deselectOnClickaway: true,
+    displayRowCheckbox: true,
+    multiSelectable: false,
+    preScanRows: true,
+    selectable: true,
+    style: {},
+  };
+
+  static contextTypes = {
     muiTheme: React.PropTypes.object.isRequired,
-  },
-
-  getDefaultProps() {
-    return {
-      allRowsSelected: false,
-      deselectOnClickaway: true,
-      displayRowCheckbox: true,
-      multiSelectable: false,
-      preScanRows: true,
-      selectable: true,
-      style: {},
-    };
-  },
+  };
 
-  getInitialState() {
-    return {
-      selectedRows: this._calculatePreselectedRows(this.props),
-    };
-  },
+  state = {
+    selectedRows: this._calculatePreselectedRows(this.props),
```

Comment:
This looks like it's not supposed to be like this, only in `componentWillMount` maybe?